doc/03_reference/verification.rst

```diff
@@ -125,14 +122,9 @@ to tell the RISCV-DV code where to find them:
     export RISCV_GCC="$RISCV_TOOLCHAIN/bin/riscv32-unknown-elf-gcc"
     export RISCV_OBJCOPY="$RISCV_TOOLCHAIN/bin/riscv32-unknown-elf-objcopy"
     export SPIKE_PATH=/path/to/spike/bin
-    export OVPSIM_PATH=/path/to/ovpsim/bin
-
-(Obviously, you only need to set ``SPIKE_PATH`` or ``OVPSIM_PATH`` if
-you have installed the corresponding instruction set simulator)
 
 .. _Spike: https://github.com/riscv/riscv-isa-sim
```

Comment:
Do we want to keep this here? Given you need to use LRSpike best to remove it to avoid confusion